services/transaction/tests/functional/mysql.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const mysqlIndex = require('../../shared/indexdb/mysql');
+const schema = require('../../shared/schema/multisignature');
+
+const tableName = 'testSchemaTransaction';
+const getIndex = () => mysqlIndex(tableName, schema);
+
+const { transaction } = require('../constants/multisignature');
```

Comment:
Please consider renaming it to `transactions` since it contains a list of transactions.

```suggestion
const { transactions } = require('../constants/multisignature');
```